docs/hardware/devices/rak/core-modules.mdx

```diff
@@ -91,16 +118,15 @@ Further information on the RAK11200 can be found on the [RAK Documentation Cente
     - 915 MHz
     - 920 MHz
     - 923 MHz
-  - **Connectors**
-    - U.FL antenna
+  - **Connectors:**
+    - U.FL/IPEX antenna connector for LoRa
 
-Further information on the RAK13300 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK13300/Overview/#product-description).
+### RAK11310 Resources
 
-<img
-  alt="RAK4631 5005 11200"
-  src="/img/hardware/rak11200.jpg"
-  style={{ zoom: "50%" }}
-/>
+- Firmware file: `firmware-rak11310-X.X.X.xxxxxxx.uf2`
+- Further information on the RAK13300 can be found on the [RAK Documentation Center](https://docs.rakwireless.com/Product-Categories/WisBlock/RAK13300/Overview/#product-description).
```

Comment:
Hmm, this mentions the RAK**13300**, which is the LoRa module to be used with the RAK11200 (that is already listed there). The link should be https://docs.rakwireless.com/Product-Categories/WisBlock/RAK11310/Overview/#product-description

Reply:
Nah, it's just too confusing. RAK should really reconsider their naming scheme